### PR TITLE
add messages for more events

### PR DESCRIPTION
**Pull request by Panaetius**

### CODE REVIEW

File: components/renku_data_services/message_queue/avro_models/io/renku/events/v1/project_authorization_added.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                "type": "string"
+            },
+            {
+                "name": "role",
+                "type": "io.renku.events.v1.ProjectMemberRole"
+            }
+        ]
+    }"""
+
+    def serialize_json(self) -> str:
```
Comment (mana-alisafaee) [inline components/renku_data_services/message_queue/avro_models/io/renku/events/v1/project_authorization_added.py:42]: This method and the following two repeat in these classes. Should we make a super class and factor them into that class?

File: components/renku_data_services/project/db.py [hunk 2]
```diff
@@ -113,13 +114,20 @@ async def insert_project(self, user: base_models.APIUser, project: models.Projec
         project_orm = schemas.ProjectORM.load(project)
         project_orm.creation_date = datetime.now(timezone.utc).replace(microsecond=0)
         project_orm.created_by = user.id
+        match project_orm.visibility:
+            case Visibility.private | Visibility.private.value:
+                vis = MsgVisibility.PRIVATE
+            case Visibility.public | Visibility.public.value:
+                vis = MsgVisibility.PUBLIC
+            case _:
+                raise NotImplementedError(f"unknown visibility:{project_orm.visibility}")
```
Comment (mana-alisafaee) [inline components/renku_data_services/project/db.py:123]: Python is getting too advanced 😄!

**Review by mana-alisafaee — COMMENTED** [review]
Great! Please see my comment.

File: components/renku_data_services/message_queue/avro_models/io/renku/events/v1/project_authorization_added.py (hunk 1 above)
Comment (Panaetius) [reply components/renku_data_services/message_queue/avro_models/io/renku/events/v1/project_authorization_added.py:42]: everything in message_queue/avro_models is generated code

**Review by mana-alisafaee — APPROVED** [review]
Thank you!

**Pull request merged**